From 28430ac46d7b319795885dbc554926747b75d04c Mon Sep 17 00:00:00 2001
From: Patrick Ohly <patrick.ohly@intel.com>
Date: Fri, 21 Jul 2017 17:15:51 +0200
Subject: [PATCH] disable service and backend

This is a minimal patch which disables building the service and
backend. This makes it possible to use the normal libpolkit-gobject
together with a service replacement like groupcheck.

Upstream-Status: Inappropriate [Embedded]

Signed-off-by: Patrick Ohly <patrick.ohly@intel.com>
---
 configure.ac    | 6 ------
 src/Makefile.am | 3 ++-
 2 files changed, 2 insertions(+), 7 deletions(-)

diff --git a/configure.ac b/configure.ac
index 21590b2..5774876 100644
--- a/configure.ac
+++ b/configure.ac
@@ -134,12 +134,6 @@ AC_ARG_WITH(mozjs, AS_HELP_STRING([--with-mozjs=@<:@mozjs185/mozjs-17.0|auto@:>@
             with_mozjs=auto)
 AS_IF([test x${with_mozjs} != xauto], [
   PKG_CHECK_MODULES(LIBJS, ${with_mozjs})
-], [
-  PKG_CHECK_MODULES(LIBJS, [mozjs185], have_mozjs185=yes, have_mozjs185=no)
-  AS_IF([test x${have_mozjs185} = xno], [
-  PKG_CHECK_MODULES(LIBJS, [mozjs-17.0], have_mozjs17=yes,
-   [AC_MSG_ERROR([Could not find mozjs185 or mozjs-17.0; see http://ftp.mozilla.org/pub/mozilla.org/js/])])
-  ])
 ])
 AC_SUBST(LIBJS_CFLAGS)
 AC_SUBST(LIBJS_LIBS)
diff --git a/src/Makefile.am b/src/Makefile.am
index 09fc7b3..be7cd72 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -1,5 +1,6 @@
 
-SUBDIRS = polkit polkitbackend polkitagent programs
+SUBDIRS = polkit
+# polkitbackend polkitagent programs
 
 if BUILD_EXAMPLES
 SUBDIRS += examples
-- 
2.11.0

